### ISSUE TITLE
Replacing specified fields in inheritance

### ISSUE THREAD

**Issue by sydneybrokeit** [hschreck/Aura#11]
It may be worthwhile to allow a way to rename specified fields in inheritance (e.g., "Firmware version" could be renamed to "IOS version").  Is this feasible?

**Comment by 128keaton** [comment]
@hschreck yeah, I can look at it on Wednesday (I'm trying to see if I can come in then).

**Comment by 128keaton** [comment]
I think this would be best suited for Ruby

**Comment by sydneybrokeit** [comment]
I'm inclined to agree.  It will be added in the 2.0 release (Ruby).
